CRM/Cdntaxreceipts/Form/ViewTaxReceipt.php

```diff
@@ -299,12 +291,18 @@ function postProcess() {
 
     $buttonName = $this->controller->getButtonName();
 
+    //CRM-1820 Once the receipt has been cancelled and user wants to preview or issue "Replace Receipt"
+    if ($this->_reissue && $this->_isCancelled && ($buttonName == '_qf_ViewTaxReceipt_submit' || $buttonName == '_qf_ViewTaxReceipt_next')){
+      list($receipt_number, $receipt_id) = CRM_Canadahelps_TaxReceipts_Receipt::receiptNumber($contribution->id,TRUE);
+      $contribution->cancelled_replace_receipt_number  = $receipt_number;
```

Comment:
I believe this was renamed... can you double check?

Reply:
Once the receipt has been cancelled there would be 2 buttons

- '_qf_ViewTaxReceipt_submit' is for Preview

- '_qf_ViewTaxReceipt_next' is for Replace Receipt

We are displaying Cancelled receipt number text **Cancels and Replaces Receipt #:** while both actions.